### DISCUSSION TITLE
Using Bootstrap 4  in angular 9

### DISCUSSION THREAD

**Discussion by Rezaalikhani71** [twbs/bootstrap#31453] [General] [answered]
Why everything show left side in the page

**Comment by Johann-S** [comment, marked as answer]
Sorry, but we cannot help with how-to or general troubleshooting issues here; only bug reports and feature requests. You'll want to ask around in the official Slack team or Stack Overflow. [See our readme for more details](https://github.com/twbs/bootstrap#community).
